app/controllers/sessions_controller.rb

```diff
@@ -155,4 +163,9 @@ def setup_mfa_authentication
     return if @user.mfa_disabled?
     session[:mfa_user] = @user.id
   end
+
+  def session_active?
+    return true if session[:mfa_expires_at].nil?
```

Comment:
I think we can remove this line since `session[:mfa_expires_at]` will always be set, or least not swallow it up and return false.

Reply:
This is more needed for the tests to not all fail out. This is because with the normal user path, `create` gets called first which initializes it. But in a lot of the tests, this does not get called first so `session[:mfa_expires_at]` will be nil.

Reply:
Ah ok, in that case, I would rather have the tests be more accurate and set the `session[:mfa_expires_at]` in the test cases.